cpp/src/backends/process/model_worker.cc

```diff
@@ -1,5 +1,8 @@
 #include "src/backends/process/model_worker.hh"
 
+namespace fs = std::filesystem;
```

Comment:
not sure if we need this, I see this is also used in logging_test.cc, wondering if there's a way to define this at single place.